lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -558,6 +558,11 @@ class FatMemManager : public MemManagerCore {
                     m_slot1.zeroedMemory());
   }
 
+  FatMemValTy setMemory(unsigned int val) const {
```

Comment:
is this correct? I am not sure what `setMemory` supposed to do, but it is strange that all slots are rewritten using the same value